hyperspy/_signals/signal1d.py

```diff
@@ -1031,16 +1031,32 @@ def filter_butterworth(self,
 
     def _remove_background_cli(
             self, signal_range, background_estimator, fast=True,
-            zero_fill=False, show_progressbar=None):
-        from hyperspy.models.model1d import Model1D
-        model = Model1D(self)
-        model.append(background_estimator)
+            zero_fill=False, show_progressbar=None, model=None,
+            return_model=False):
```

Comment:
Shouldn't this have a docstring? I know it's hidden, but would improve readability/maintainability.